aaronpoweruser.ReadwiseUnofficial/plugin.json

```diff
@@ -0,0 +1,88 @@
+{
+  "COMMENT": "Details on these fields: https://help.noteplan.co/article/67-create-command-bar-plugins",
+  "macOS.minVersion": "10.13.0",
+  "noteplan.minAppVersion": "3.4.0",
+  "plugin.id": "aaronpoweruser.ReadwiseUnofficial",
+  "plugin.name": "📚 Readwise Unofficial",
+  "plugin.version": "0.1.0",
+  "plugin.lastUpdateInfo": "First release",
+  "plugin.description": "A sync plugin for readwise",
+  "plugin.author": "aaronpoweruser",
+  "plugin.dependencies": [],
+  "plugin.script": "script.js",
+  "plugin.url": "https://github.com/NotePlan/plugins/blob/main/aaronpoweruser.ReadwiseUnofficial/README.md",
+  "plugin.changelog": "https://github.com/NotePlan/plugins/blob/main/aaronpoweruser.ReadwiseUnofficial/CHANGELOG.md",
+  "plugin.commands": [
+    {
+      "name": "Readwise sync",
+      "description": "Sync readwise highlights",
+      "jsFunction": "readwiseSync"
+    },
+    {
+      "name": "onOpen",
+      "description": "DO NOT EDIT THIS COMMAND/TRIGGER",
+      "jsFunction": "onOpen",
+      "hidden": true
+    },
+    {
+      "name": "onEditorWillSave",
+      "description": "DO NOT EDIT THIS COMMAND/TRIGGER",
+      "jsFunction": "onEditorWillSave",
+      "hidden": true
+    }
+  ],
+  "plugin.settings": [
+    {
+      "COMMENT": "Plugin settings documentation: https://help.noteplan.co/article/123-plugin-configuration",
+      "type": "heading",
+      "title": "Readwise Unofficial Settings"
+    },
+    {
+      "title": "Access token",
+      "key": "accessToken",
+      "required": true,
+      "type": "string",
+      "description": "Enter your readwise access token from readwise.io/access_token",
+      "default": ""
+    },
+    {
+      "title": "Folder name",
+      "key": "baseFolder",
+      "type": "string",
+      "description": "Enter some string and see it change when the plugin is run",
```

Comment:
I don't understand why you'd want the Folder name to change?

Reply:
I made this for myself. I want my book notes under my PARA system so I set it to "Resources/readwise"

Reply:
I also use PARA so that's where I'd like it as well. But the description makes it sound like it will change each time you run it ...?